FirstProject/assets/login.less

```diff
@@ -0,0 +1,73 @@
+//Login.less
+
+body {
+  background-color:#fafafa;
+}
+
+header {
+  background: url('../img/feedback-banner.jpg') center center no-repeat;
+  background-size: cover;
+  height: 600px;
+
+  .grid {
+    height: 100%;
+  }
+}
+
+.login-form {
+  background-color: #ffffff;
+  align-self: center;
+  text-align: center;
+  padding-bottom: 30px;
+
+  .login-form-header {
+    line-height: 60px;
+    background-color: #03A9F4;
+    color: #ffffff;
+    font-weight: bold ;
+    text-align: center;
+  }
+
+  .login-form-content  {
+    padding:15px;
+    input {
+      width: 100%;
+      border: none;
+      border-bottom: 1px solid #cccccc;
+      margin-top: 25px;
+    }
+  }
+
+  .login-form-submit {
+
+  }
+}
+
+
+.login-section-1 {
```

Comment:
I'm not saying you can't, I'm just saying there are possibly other (more modern) ways of achieving the same thing, e.g. using `:nth-child`. This might not apply in every case, so every once in a while, you might need to do something like this, but I'm just trying to make sure you're aware of your options and not simply polluting your markup with extra classes because it's the simplest solution available.